build-farm/platform-specific-configurations/windows.sh

```diff
@@ -24,7 +24,6 @@ export DRAGONWELL8_BOOTSTRAP=/cygdrive/C/openjdk/dragonwell-bootstrap/jdk8u272-g
 export ALLOW_DOWNLOADS=true
 export LANG=C
 export OPENJ9_NASM_VERSION=2.13.03
-export OPENSSL_VERSION=1.1.1m
```

Comment:
Note sure about this change - was that variable only used by OpenJ9?

Reply:
Yes, as far as I can tell it was only inside `if openj9`'s.

Reply:
Windows will now use [this line](https://github.com/adoptium/temurin-build/blob/36ce3640b6d5317b2282c83892ec6db7731dc170/sbin/prepareWorkspace.sh#L267) like all other platforms.

Reply:
To be clear, it's the use of this variable that's within an OpenJ9 section. That variable  definition in the environment should not affect Temurin which does not use OpenSSL.

If you have a concern then I suppose the definition could be moved within another `if` block but we do have other variables outside such blocks which are not used by Temurin builds.

Reply:
Yes, I think it was only used by OpenJ9. I don't see any occurrences of `OPENSSL_VERSION` in this repo after this change.